src/app/static/src/app/components/plots/timeSeries/downloadTimeSeries/DownloadTimeSeries.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div id="indicator-download">
+        <download-button
+            :name="'downloadIndicator'"
+            :disabled="!hasData"
+            @click="download"></download-button>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import DownloadButton from "./DownloadButton.vue";
+import {computed} from "vue";
+import {useStore} from "vuex";
+import { RootState } from '../../../../root';
+import {appendCurrentDateTime} from "../../../../utils";
+import {exportService} from "../../../../dataExportService";
+import {debounce_leading} from "../../utils";
+
+const store = useStore<RootState>();
+
+const unfilteredData = computed(() => {
+    const dataSource = store.state.plotSelections.timeSeries.controls.find(c => c.id === "time_series_data_source")?.selection[0].id;
+    if (!dataSource) {
+        return undefined
+    }
+    return store.state.reviewInput.datasets[dataSource]?.data
```

Comment:
ahhh is this another place we are storing these? ive been using the `surveyAndProgram` state with `anc` and `program` keys to check if these exist, is this data being saved in two places? or are those slightly different?

Reply:
I think surveyAndProgram is the raw data right? Used for the maps, but this `reviewInput` is the aggregated time series data